experimenter/experimenter/experiments/api/v5/inputs.py

```diff
@@ -46,6 +56,9 @@ class ExperimentInput(graphene.InputObjectType):
     countries = graphene.List(graphene.String)
     documentation_links = graphene.List(DocumentationLinkInput)
     excluded_experiments = graphene.List(graphene.NonNull(graphene.Int))
+    excluded_experiments_branches = graphene.List(
+        NimbusExperimentBranchThroughExcludedInput
+    )
```

Comment:
This generates `(NimbusExperimentBranchThroughExcludedInput | null)[] | null;`

Lets get rid of some of those nulls:

```suggestion
    excluded_experiments_branches = graphene.List(
        graphene.NonNull(NimbusExperimentBranchThroughExcludedInput)
    )
```